python/src/open_discourse/01_data/01_download_raw_data.py

```diff
@@ -32,22 +32,50 @@
     "https://www.bundestag.de/resource/blob/870686/91b713c492499db98eec5b2f8f142d20/pp19.zip",
 ]
 
-for link in tqdm(zip_links, desc="Download & unzip election period data..."):
-    # Extract election period from URL
-    electoral_term_str = "electoral_term_" + regex.search(r"pp(\d+).zip$", link).group(
-        0
-    )
 
-    print(f"Download & unzip '{electoral_term_str}'...", end="", flush=True)
+class MockLogger:
```

Comment:
why do we need a new `class`? I think @erwee6 already created a helper method for logging that creates a new method so we should use it. Also, you create the same class in every file, that is definitely **not** what we should do ever

Reply:
and I don't understand the meaning of this class honestly, it is not real logging but basically still uses print? I think we should not use such a workaround that in the end still uses print, we already agreed to switch to a proper logging and that is already implemented as I said by Robert

Reply:
that class is meant as a placeholder for Robert's logger, from my understanding the PR of Robert with the logger is not merged yet?

Reply:
Yes, that is absolutely correct but you would normally never implement a placeholder if a solution is already a PR and in review, just wait ;) The PR should actually be merged already I don't have much more to say but let's discuss in the next meeting. But a placeholder would be insane at the moment when Robert's solution is already in review and as such close to be merged in main as this means you need to refactor and remove your code 1-2 days or weeks after you implemented it ;) In my opinion this is just waste of your time